gulpfile.js

```diff
@@ -3,43 +3,44 @@ var gulp = require('gulp'),
   pug = require('gulp-pug'),
   config = require('./config.json'),
   browserSync = require('browser-sync').create(),
-  paths = config.paths;
+  paths = config.paths,
+  path = require('path');
 
 gulp.task('default', ['build-sass', 'build-views', 'serve', 'watch']);
 
-gulp.task('watch', function(){
-  gulp.watch('./scss/**/*.scss', ['build-sass']);
-  gulp.watch('./views/*.pug', ['build-views']);
+gulp.task('watch', function () {
+  gulp.watch(`${paths.source.scss}/**/*.scss`, ['build-sass']);
+  gulp.watch(`${paths.source.pug}/*.pug`, ['build-views']);
 });
 
 gulp.task('serve', function () {
   browserSync.init({
     server: {
-      baseDir: './public',
+      baseDir: paths.dest.baseDir,
       directory: true
     },
-    files: ['./public/**/*']
+    files: [`${paths.dest.baseDir}/**/*`]
```

Comment:
`path.join` works for globs as well: https://runkit.com/vollemelk/path-join-example